ddtrace/contrib/internal/openai/utils.py

```diff
@@ -116,7 +116,7 @@ def _loop_handler(span, chunk, streamed_chunks):
     When handling a streamed chat/completion/responses,
     this function is called for each chunk in the streamed response.
     """
-    if span.get_tag("openai.response.model") is None:
+    if span.get_tag("openai.response.model") is None or not span.get_tag("openai.response.model"):
```

Comment:
Instead of checking both cases, can't we just replace the `is None` check with `not span.get_tag("...")`?